modal/_asgi.py

```diff
@@ -23,8 +23,6 @@ async def receive():
 
         # Run the ASGI app, while draining the send message queue at the same time,
         # and yielding results.
-        # TODO(gongy): we currently create one ASGI instance per concurrent input,
-        # but it would be sufficient to share this background ASGI context.
```

Comment:
I removed this comment because I think it's outdated — we only call the function once in `import_function()` to produce the ASGI app right now even for multiple inputs and regardless of input concurrency. Then this gets passed into `asgi_app_wrapper`. (cc @gongy)